docs/gallery.md

```diff
@@ -2,32 +2,90 @@
 
 A list of existing Wegue projects.
 
-[Architect Oeds de Leeuw Wieland](https://bevinggevoeld.nl/oeds/groot/
-)
+---
 
-![](_media/screenshot_bevinggevoeld.jpg)
+## [Time-Travel through Rhineland-Palatinate](https://apps.meggsimum.de/wegue-demos/rlp-historische-tk/?year=1960)
 
+This demo shows how to interact with time-dependent geodata. It visualizes topographic maps of Rhineland-Palatinate in Germany from 1885 until 2020. The data is provided by [Landesamt für Vermessung und Geobasisinformation
```

Comment:
Maybe better `from 1885 to 2020`?

Reply:
I think both `to` and  `until` is  possible

Reply:
At least that's  what I could quickly find in the internet. That's why I would keep it.